src/extension/ln/nostr/index.ts

```diff
@@ -61,19 +80,46 @@ export default class NostrProvider {
           !messageEvent.data ||
           !messageEvent.data.response ||
           messageEvent.data.application !== "LBE" ||
-          messageEvent.data.scope !== "nostr"
+          messageEvent.data.scope !== "nostr" ||
+          !window.nostr._requests[messageEvent.data.id]
         ) {
+          if (
+            messageEvent.data &&
+            messageEvent.data.response &&
+            messageEvent.data.application === "LBE" &&
+            messageEvent.data.scope === "nostr" &&
+            !window.nostr._requests[messageEvent.data.id]
+          ) {
+            console.log(
```

Comment:
I'm seeing events here that already have been removed from the `_requests` object:  

![image](https://user-images.githubusercontent.com/100827540/225454182-38ce38aa-4e2f-4b56-bff1-00c33da13daf.png)

Any idea where this might come from? They seem to be called twice. 🤔

Reply:
Seems those event handlers are not cleaned up if aborted early. (`nostr call already executing`)

![image](https://user-images.githubusercontent.com/100827540/225459940-857ae060-c719-4265-82cf-a4608d4c4f5f.png)